ratereich/localisation/spain_l_english.yml

```diff
@@ -0,0 +1,4 @@
+﻿l_english:
+  SPR_market_liberalism:0 "Iberian Federation"
+  CAN_right_authoritarianism_DEF:0 "Wallonia"
+  CAN_right_authoritarianism_ADJ:0 "Walloon"
```

Comment:
Walloon? For canada's right- ?

Reply:
I just copy pasted Wallonia's file and edited it for Spain, I left it there without noticing